src/Sylius/Bundle/ShopBundle/Resources/views/Product/macros.html.twig

```diff
@@ -4,3 +4,23 @@
     {% endif %}
     <span> {{ sylius_calculate_price(variant)|sylius_price }} </span>
 {% endmacro %}
+
+{% macro list(products, limit = 2) %}
```

Comment:
If _singleProduct element on page will have small portion of information and little pictures then you may want to put more than 2 element in one row. Everything depends on  _singleProducts final form, becouse now it has only the most necessary fields.